### PR TITLE
Migrate from jkremser -> kedify org

### PR DESCRIPTION
**Pull request by jkremser**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Teams:
+* @kedify/all
```
Comment (jkremser) [inline CODEOWNERS:2]: this probably requires some clicking in web ui @zroubalik 🥺

Comment (jkremser) [reply CODEOWNERS:2]: I would also like to add a gh token for creating the PRs to this repo if there is a release (the krew manifest needs to be updated after the release), but I can't see the settings page above the repo

Comment (zroubalik) [reply CODEOWNERS:2]: Will try to make you admins asap

Comment (zroubalik) [reply CODEOWNERS:2]: Done

**Pull request merged**